tools/install.sh

```diff
@@ -262,4 +264,9 @@ else
     exit 1
 fi
 
-echo "Installation complete."
+echo "The installation of LORIS software components is now complete."
```

Comment:
needs to be changed since the installation of software components happens *after* during `make`

Reply:
- [x] done : see my [commit 475ca8](https://github.com/aces/Loris/pull/5205/commits/475ca805073aba4296b34ac0698d3e16e4a5d3fa)